src/components/collapsible/Collapsible.js

```diff
@@ -50,11 +55,18 @@ class Collapsible extends React.Component<Props, State> {
 
     constructor(props: Props) {
         super(props);
+        this.ref = React.createRef();
         this.state = {
             isOpen: props.isOpen,
         };
     }
 
+    componentDidMount() {
+        if (this.ref.current) {
+            this.ref.current.style.height = `0px`;
```

Comment:
Could we instead we set the height value in state on `componentDidMount` when `isOpen` changed, then pass the height style to the wrapping div in render?

Reply:
Yeah this was causing were you couldn't default a collapsible to open.